### DISCUSSION TITLE
Despite claim Antmedia does not install on Mac OS.

### DISCUSSION THREAD

**Discussion by mooncatventures-group** [ant-media/Ant-Media-Server#5699] [Q&A]
The website and ads claims this product can be installed on Mac OS  

After trying several competitors and being disappointed with features , we decided to give AntMeda a try, foolishly we went for the paid trial instead of opting for the community version because we didn't want to waste our time configuring the product. We need a good WebRTC server, 
After many attempts, including making alterations to the script for clearly Linux-specific directories  and functions, We used brew to install Apget for instance, we found we still couldn't get past the incredibly poor information on Red 5 installation.

Frustrating we even put in a support request. The service level agreement says 24 hours max 3 days, its been over 4 and no reply.

We wasted precious hours on a product that simply does not wor as advertised.. And promises non-existent support for paid customers. Very disappointed, unless you are on vanilla Ubuntu Linux stay far away from this product.

**Comment by muratugureminoglu** [comment]
Hi @mooncatventures-group 

Sorry for our late reply. I have answered your questions in GitHub issues, but let me explain the situation once again. In previous days, we sent an automatic message stating that there might be delays in responses because we were undergoing internal training for all our customers. So, this might be in your spam folder.

Kindly reminder, the support channel you're using is not for paid support, it's only for enterprise users. Paid support is different. 

Regarding your question, unfortunately, the installation script does not work on MacOS. We officially support Ubuntu, and you can also install it on other Linux versions like RockyLinux, CentOS, AlmaLinux, and RHEL. However, you can try the steps mentioned [here](https://community.antmedia.io/topic/746-ant-media-server-ce-on-imac/) to run it on MacOS?

By the way, I recommend using [docker](https://antmedia.io/docs/guides/clustering-and-scaling/docker/docker-and-docker-compose-installation/) for a smooth installation.

If you have any further questions, please do not hesitate to ask.
